src/main/java/org/broadinstitute/hellbender/engine/spark/SparkContextFactory.java

```diff
@@ -21,11 +21,13 @@
     public static final Map<String, String> TEST_ATTRIBUTES = ImmutableMap.<String, String>builder()
             .put("spark.ui.enabled", Boolean.toString(SPARK_DEBUG_ENABLED))
             .put("spark.kryoserializer.buffer.max", "256m")
+            .put("spark.hadoop.fs.file.impl.disable.cache", "true") // so RawLocalFileSystem is not cached between tests
             .build();
 
     public static final Map<String, String> MANDATORY_ATTRIBUTES = ImmutableMap.<String,String>builder()
             .put("spark.serializer", KryoSerializer.class.getCanonicalName())
             .put("spark.kryo.registrator", "org.broadinstitute.hellbender.engine.spark.GATKRegistrator")
+            .put("spark.hadoop.fs.file.impl", "org.apache.hadoop.fs.RawLocalFileSystem") // avoid writing CRC files for local files
```

Comment:
It remaps _file://_ URIs to use Hadoop's `RawLocalFileSystem` implementation rather than `LocalFileSystem`. `LocalFileSystem` writes CRC files, whereas `RawLocalFileSystem` does not.

I'll expand the comment in the code.